src/store/slices/playlists/slice.ts

```diff
@@ -33,8 +35,10 @@ string,
         const playlistName = payload
         try {
             const id = await firestoreApi.createPlaylist(currentUser!.id, currentUser!.name, playlistName)
+            thunkApi.dispatch( notificationActions.ADD_NOTIFICATION({message: "playlist_created", severity: SEVERITY.Info}))
```

Comment:
instead of dispatch a notification action here(and all the notification action), you can add extrareducers in the notification slice.
for example:
[playlistsAsyncActions.createPlaylist.fulfilled.type]: (state) => {
      state.notification.push({
        id: getId(),
        message: 'playlist_created'
        severity: Severity.SUCCESS,
      })
with this method, the playlist slice will be much cleaner, and all the notification action will be handled in the notification slice.

Reply:
thanks, fixed!